linode/manager
# refactor: [M3-6063] - Refactor components to use TypeToConfirmDialog component

## Description 📝
Refactor components to standardize type-to-confirm implementation using the `TypeToConfirmDialog` component.

## Major Changes 🔄
- Introduced the following new `types` in the `EntityInfo` interface: `Database`, `Kubernetes`, `AccountSetting`.
- Introduced the `subType` property in the `EntityInfo` interface: `Backup`, `Cluster`, `ObjectStorage`, `CloseAccount`.

## How to test 🧪
1. Verify type-to-confirm functionality across the application.
2. Check for any visual regressesions.
